### PR TITLE
Not creating "Accept:null" header for Java-Jersey2 generator

### PR DESCRIPTION
**Pull request by S2021Git**
Before the fix, if using Jersey2 as generator and including DELETE operations that don't "accept" any body, the generated code executes HTTP requests with header "Accept:null", which is not a valid header.
After the fix, it will not create "Accept:null" as the header.

This is to fix issue #6882
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (5.3.0), `6.0.x`
- [ ] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache [hunk 1]
```diff
@@ -1190,7 +1190,12 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
       }
     }
 
-    Invocation.Builder invocationBuilder = target.request().accept(accept);
+    Invocation.Builder invocationBuilder;
+    if (accept != null) {
+      invocationBuilder = target.request().accept(accept);
+      } else {
+      invocationBuilder = target.request();
+      }
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache:1198]: Thanks for the PR. Looks like this line and line 1196 are not indented correctly.

Comment (S2021Git) [reply modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache]: @wing328  Hi thank you for the review. I just made the update, can you please take a look. Thank you.

**Comment by wing328** [comment]
cc @bbdouglas (2017/07) @sreeshas (2017/08) @jfiala (2017/08) @lukoyanov (2017/09) @cbornet (2017/09) @jeff9finger (2018/01) @karismann (2019/03) @Zomzog (2019/04) @lwlee2608 (2019/10)

**Comment by wing328** [comment]
LGTM. Thanks for the fix.

**Pull request merged**